interactive/submit.py

```diff
@@ -27,38 +27,38 @@
         --output-dir output/{ID}
     outputs:
       moderately_sensitive:
-        table: output/*/counts_per_*.csv
-        list_sizes: output/*/list_sizes.csv
+        table: output/{ID}/counts_per_*.csv
+        list_sizes: output/{ID}/list_sizes.csv
 
-  generate_measures:
+  measures_{ID}:
     run: python:latest python analysis/generate_measures.py output/{ID}
-    needs: [generate_codelist_report]
+    needs: [codelist_report_{ID}]
     outputs:
       moderately_sensitive:
-        measure: output/*/measure_counts_per_week_per_practice.csv
+        measure: output/{ID}/measure_counts_per_week_per_practice.csv
 
-  generate_top_5_table:
+  top_5_table_{ID}:
     run: python:latest python analysis/top_codes_table.py output/{ID}
-    needs: [generate_codelist_report]
+    needs: [codelist_report_{ID}]
     outputs:
       moderately_sensitive:
-        table: output/*/top_5_code_table.csv
+        table: output/{ID}/top_5_code_table.csv
 
-  generate_deciles_charts:
+  deciles_charts_{ID}:
     run: >
       deciles-charts:v0.0.24
-        --input-files output/measure_counts_per_week_per_practice.csv
+        --input-files output/{ID}/measure_counts_per_week_per_practice.csv
         --output-dir output/{ID}
     config:
       show_outer_percentiles: false
       tables:
         output: true
       charts:
         output: true
-    needs: [generate_measures]
+    needs: [measures_{ID}]
     outputs:
       moderately_sensitive:
-        deciles_charts: output/*/deciles_*_*.*
+        deciles_charts: output/{ID}/deciles_*.*
```

Comment:
Is this change going to cause any unexpected issues?

Reply:
Hah, that's a fun phrasing - I don't know, as they'd be unexpected! :joy: 

I think it is more explicit and more correct, but should make no difference.

Reply:
> `deciles_*.*` is less explicit than `deciles_*_*.*`?

It's actually less, as we are not requiring a middle `_` character